lib/generators/outbox/templates/migration.rb

```diff
@@ -0,0 +1,13 @@
+class OutboxCreate<%= table_name.camelize.singularize %> < ActiveRecord::Migration<%= migration_version %>
```

Comment:
```suggestion
class CreateOutbox<%= table_name.camelize.singularize %>Table < ActiveRecord::Migration<%= migration_version %>
```
?

Reply:
We tried following Devise's naming convention for migrations, but I like this as well.